.github/settings.yml

```diff
@@ -1,12 +1,17 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
 repository:
   name: firefly-tezosconnect
+  description: Firefly Tezos Connect
+  homepage: https://github.com/hyperledger/firefly-tezosconnect
   default_branch: main
-  has_downloads: false
+  has_downloads: true
   has_issues: true
   has_projects: true
-  has_wiki: false
+  has_wiki: true
```

Comment:
Do we have content for a Tezosconnect specifici wiki? Typically we've been steering away from repo-specific Wikis and putting everything in the main FireFly docs site.

Reply:
I've found this option for EVM Connect and thought that it's about the main FireFly docs site.
But now I realised that it's about specific project wiki.
Thanks for that. (The option needs to be updated for EVM Connector as well)